app/bundles/MessengerBundle/Translations/en_US/messages.ini

```
@@ -1,23 +1,14 @@
-mautic.config.tab.messengerconfig="Messenger Settings"
-mautic.messenger.config.enabled="Do you want to send messages (emails, sms, push) via a queue?"
-mautic.messenger.config.enabled.true="Yes"
-mautic.messenger.config.enabled.false="No, I want to send messages immediately"
-mautic.messenger.config.enabled.tooltip="Messages are sent via a queue so that the system doesn't slow down. This means that if you have a lot of messages to send, it could take a while to send them all."
-mautic.messenger.config.transport="Queue transport type"
-mautic.messenger.config.transport.tooltip=""
-mautic.messenger.config.host="Host"
-mautic.messenger.config.host.tooltip="Host"
-mautic.messenger.config.port="Port"
-mautic.messenger.config.port.tooltip="Port"
-mautic.messenger.config.user="Username"
-mautic.messenger.config.user.tooltip="Username"
-mautic.messenger.config.password="Password"
-mautic.messenger.config.password.tooltip="Password"
+mautic.config.tab.messengerconfig="Queue Settings"
 mautic.messenger.config.retry_strategy="Retry strategy"
 mautic.messenger.config.retry_strategy.max_retries="Max retries"
 mautic.messenger.config.retry_strategy.delay="Delay"
 mautic.messenger.config.retry_strategy.multiplier="Multiplier"
 mautic.messenger.config.retry_strategy.max_delay="Max delay"
-mautic.messenger.config.transport.doctrine="Database Queue"
-
-
+mautic.messenger.config.dns_help_general="Queuing is not enabled by default (scheme is set to 'sync'). If you want to start using a queue, please follow the documentation on <a href="%link%">%link%</a>."
```

```suggestion
mautic.messenger.config.dns_help_general="Queuing is not enabled by default (scheme is set to 'sync'). If you want to start using a queue, please follow the documentation at <a href="%link%">%link%</a>."
```

Should we really be sending the end-user to Symfony documentation here (as rendered in the screenshots provided in Slack)? I wonder if it is more helpful to send them to a Mautic-specific documentation resource?

---

We provided Symfony documentation links as we didn't have links to Mautic docs. We will address this in a follow-up PR once Mautic documentation page is available.